packages/web-components/scripts/addPreactAliases.js

```diff
@@ -0,0 +1,42 @@
+import { readFile, writeFile } from "node:fs/promises";
+import path from "node:path";
+import { createRequire } from "node:module";
+
+const require = createRequire(import.meta.url);
+
+const addPreactAliases = async () => {
+  // Determine the tsconfig.json location (assumed to be in the project root)
+  const tsconfigPath = path.join(process.cwd(), "tsconfig.json");
```

Comment:
You can use just `path.resolve("tsconfig.json")` because it returns an absolute path to the file using the process' `cwd` as the base path